jenkins/ompi/Dockerfile.centos7

```diff
@@ -0,0 +1,46 @@
+ARG OMPI_CI_OS_NAME=centos
+ARG OMPI_CI_OS_VERSION=7
+FROM ${OMPI_CI_OS_NAME}:${OMPI_CI_OS_VERSION}
+
+RUN yum install -y \
+    atk \
+    bc \
+    binutils-devel \
+    bzip2 \
+    cairo \
+    environment-modules \
+    flex \
+    gcc \
+    gcc-c++ \
+    gcc-gfortran \
+    git \
+    gtk2 \
+    jq \
+    libmnl \
+    libnl3 \
+    make \
+    numactl-devel \
+    numactl-libs \
+    perl \
+    perl-Data-Dumper \
+    perl-Thread-Queue \
+    sudo \
+    tcsh \
+    tk \
+    valgrind-devel \
+    wget
+
+# Engineering versions of Mellanox HPC-X components are loaded from the Mellanox lab environment modules
+RUN echo "/hpc/local/etc/modulefiles" >> /usr/share/Modules/init/.modulespath
+
+ARG OMPI_CI_MOFED_VERSION=4.7-1.0.0.1
+# TODO: 'rhel7.6' is hardcoded for centos7
+ENV MOFED_DIR MLNX_OFED_LINUX-${OMPI_CI_MOFED_VERSION}-rhel7.6-x86_64
+ENV MOFED_SITE_PLACE MLNX_OFED-${OMPI_CI_MOFED_VERSION}
+ENV MOFED_IMAGE MLNX_OFED_LINUX-${OMPI_CI_MOFED_VERSION}-rhel7.6-x86_64.tgz
+RUN wget --no-verbose http://content.mellanox.com/ofed/${MOFED_SITE_PLACE}/${MOFED_IMAGE} && \
```

Comment:
Why do you pull an OFED and not an HPCX tarball?

Reply:
HPCX components are currently used from `module load hpcx-gcc-stack`. As we discussed verbally we can consider changing it to released HPC-X as a future improvement.